packages/ripple-binary-codec/src/types/currency.ts

```diff
@@ -25,11 +25,6 @@ function isIsoCode(iso: string): boolean {
 
 function isoCodeFromHex(code: Buffer): string | null {
   const iso = code.toString()
-  if (iso === 'XRP') {
-    throw new Error(
```

Comment:
This should probably `return null` instead of erroring

Reply:
I think that's still unexpected. The ledger has defenses up now at least to make it harder to scam people by having fake XRP on the ledger, and for folks doing historical checks it seems better to just be accurate rather than force them to know XRP becomes null.